### PR TITLE
Update to gatsby v3

### PR DESCRIPTION
**Pull request by nr-kkenney**
Changes for upgrading to Gatsby V3

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,34 +4,34 @@
...
     "date-fns": "^2.17.0",
     "front-matter": "^4.0.2",
-    "gatsby": "^2.32.4",
-    "gatsby-image": "^2.11.0",
-    "gatsby-plugin-manifest": "^2.12.0",
+    "gatsby": "^3.3.1",
+    "gatsby-image": "^3.3.0",
+    "gatsby-plugin-manifest": "^3.3.0",
     "gatsby-plugin-json-output": "^1.2.0",
     "gatsby-plugin-mdx": "^1.10.1",
```
Comment (aswanson-nr) [inline package.json:22]: This has a newer version, `2.3.0`

File: package.json [hunk 2]
```diff
@@ -4,34 +4,34 @@
...
     "gatsby-plugin-meta-redirect": "^1.1.1",
     "gatsby-plugin-offline": "^3.10.1",
```
Comment (aswanson-nr) [inline package.json:24]: This has a newer version as well, `4.3.0`

**Comment by roadlittledawn** [comment]
one weird thing i just noticed is on the data dictionary page. when i go to filter for a specific `Data Type` i get an error in the console and the page doesn't seem to apply the filter. https://pr-1946.d1vx4w5vgyla4s.amplifyapp.com/attribute-dictionary/

**Review by aswanson-nr — COMMENTED** [review]
Looks great! I just noticed two plugins that can be updated

File: package.json (hunk 1 above)
Comment (nr-kkenney) [reply package.json]: updated.

File: package.json (hunk 2 above)
Comment (nr-kkenney) [reply package.json]: updated.

**Comment by nr-kkenney** [comment]
resolved the issue @roadlittledawn mentioned. switched `useNavigate` to `navigate` in our import from reach/router.

**Review by zstix — APPROVED** [review]

**Pull request merged**